experimental/AlgebraicStatistics/src/GraphicalModels.jl

```diff
@@ -0,0 +1,119 @@
+export GraphicalModel, graph, ring, param_ring, param_gens, vanishing_ideal
+
+import Oscar.graph
+import Oscar.vertices
+import Oscar.gens
+import Base.show
+
+
+###################################################################################
+#
+#       Additional functions for graphs
+#
+###################################################################################
+
+
+function vertices(G::Graph)
+
+    E = [[src(e), dst(e)] for e in edges(G)]
+    
+    sort(unique(reduce(vcat, E)))
+end
+
+
+
+###################################################################################
+#
+#       Additional functions for matrices
+#
+###################################################################################
+
+
+function cofactor(A)
```

Comment:
maybe use `pseudo_inv` to compute this? Should be much faster than all the single det calls

Reply:
Thanks, will do.